### PR TITLE
Update util.js

### LINKED ISSUE #480: [VAX-1068] Prefix env var DATABASE_URL
The starter template uses an env var named DATABASE_URL that sets the URL for connecting to Postgres.

The name of this variable might easily clash with env vars of other projects. The goal of this task is to prefix that variable with ELECTRIC\_.

<sub>From [SyncLinear.com](https://synclinear.com) | [VAX-1068](https://linear.app/electric-sql/issue/VAX-1068/prefix-env-var-database-url)</sub>

### PR DESCRIPTION
**Pull request by KhurramJalil**
Fixes #480 

Updated utils.js to rename env variable from DATABASE_URL TO ELECTRIC_DATABASE_URL

### CODE REVIEW

File: examples/starter/template/db/util.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ shell.config.silent = true // don't log output of child processes
 const pgPort = fetchHostPortPG() ?? 5432
 const appName = fetchAppName() ?? 'electric'
 const DEFAULT_URL = `postgresql://postgres:password@localhost:${pgPort}/${appName}`
-const DATABASE_URL = process.env.DATABASE_URL || DEFAULT_URL
+const DATABASE_URL = process.env.ELECTRIC_DATABASE_URL || DEFAULT_URL
```
Comment (balegas) [inline examples/starter/template/db/util.js:14]: There are more places in the code that load process.env.DATABASE_URL (see startElectric.js).
Note that the change need to be applied to the pre-generated examples.

**Review by balegas — CHANGES_REQUESTED** [review]
Tthank you for your kind contribution.

Note that all PRs need to include a changeset that you can generate with `pnpm generate` (see other PRs). I'm adding that to the instructions

**Comment by KhurramJalil** [comment]
Thanks @balegas for reviewing and providing feedback.

I have committed more changes in the same PR with 11 files updated overall. Please review again and see if there is anything more to add.

**Comment by balegas** [comment]
I'm closing thins without merging because of the new changes to the starter that are coming in. Thanks for picking this up and there will be more #good-first-issues in the future.

**Pull request closed without merging**